discordia/client/Multipart.lua

```diff
@@ -0,0 +1,16 @@
+local Multipart = {}
```

Comment:
There's no reason to have a separate file and table for one function. It can be integrated into `TextChannel` as a local function.